portal/src/components/Popup/ImageCropPopup.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <Popup v-if="isShow" :is-show="isShow" class="crop-popup" :close="closePopup">
+    <vue-croppie
+      ref="croppieRef"
+      :boundary="{ width: 400, height: 300 }"
+      :viewport="{ width: width, height: height }"
+      :enable-resize="false"
+      @result="result"
+    >
+    </vue-croppie>
+    <button type="button" class="button crop" @click="cropImage()">
+      {{ $t('Crop') }}
```

Comment:
Staat deze al in legacy productie database?

Reply:
Nu wel 😄 . Wilde het doen als het goedgekeurd was